Sources/Utility/unlink.swift

```diff
@@ -8,12 +8,8 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
-
-import func libc.unlink
-import var libc.errno
+import Foundation
 
 public func unlink(_ path: String) throws {
```

Comment:
This function should be renamed, using `unlink` as a synonym for removing a tree is confusing, as that is not the POSIX behavior.